packages/nextjs-mf/README.md

```diff
@@ -134,29 +134,30 @@ You can see it in action here: https://github.com/module-federation/module-feder
 
 ```js
 // next.config.js
-const NextFederationPlugin = require('@module-federation/nextjs-mf/NextFederationPlugin');
+// either from default
+const NextFederationPlugin = require('@module-federation/nextjs-mf').default;
+// or named export
+const NextFederationPlugin = require('@module-federation/nextjs-mf').NextFederationPlugin;
```

Comment:
NIT:

```suggestion
const { NextFederationPlugin } = require('@module-federation/nextjs-mf');
```

Reply:
fair nit, a lot of eslint configs prefer destructuring anyway